wwwroot/fileupload.html

```diff
@@ -73,12 +73,39 @@ <h1 class="mt-4">File Upload</h1>
                             <div class="card-body">
-                                <iframe name="dummyframe" style="display: none;"></iframe>
-                                <form action="/api/file" id="file_form" method="post" enctype="multipart/form-data" target="dummyframe">
-                                    <label for="file">Filename:</label>
-                                    <input type="file" name="file" id="file"><br>
+                                    <iframe name="dummyframe" style="display: none;"></iframe>
+                                    <form action="/api/file" id="uploadForm" method="post" enctype="multipart/form-data" target="dummyframe" onsubmit="submitForm(event)">
+                                        <label for="file">Filename:</label>
+                                    <input type="file" name="file" id="file" multiple><br>
+                                
+                                    <label for="uploader">Uploader:</label>
+                                    <input type="text" id="uploader" name="uploader"><br>
+                                
+                                    <label for="weather">Weather:</label><br>
+                                    <input type="radio" id="sun" name="weather" value="sun">
+                                    <label for="sun">Sun</label><br>
+                                    <input type="radio" id="rain" name="weather" value="rain">
+                                    <label for="rain">Rain</label><br>
+                                    <input type="radio" id="snow" name="weather" value="snow">
+                                    <label for="snow">Snow</label><br>
+                                
+                                    <label for="dessert">Dessert:</label><br>
+                                    <input type="checkbox" id="Red Velvet Cake" name="dessert" value="Red Velvet Cake">
+                                    <label for="Red Velvet Cake">Red Velvet Cake</label><br>
+                                    <input type="checkbox" id="Peach Pie" name="dessert" value="Peach Pie">
+                                    <label for="Peach Pie">Peach Pie</label><br>
+                                    <input type="checkbox" id="Lemon Bars" name="dessert" value="Lemon Bars">
+                                    <label for="Lemon Bars">Lemon Bars</label><br>
+
+                                    <label for="country">Country:</label>
+                                    <select id="country" name="country">
+                                        <option value="usa">USA</option>
+                                        <option value="canada">Canada</option>
+                                        <option value="mexico">Mexico</option>
+                                    </select><br>
+                                
                                     <input type="submit" name="submit" value="Submit">
```

Comment:
If you are going to have required fields on the server side, we should add form validation on submit, and indicate for the different fields that a value is required. It also may be good to add a bubble indicating that an error has happened when we get a 400/500 back from the server.